src/InteractiveWindow/EditorTest/InteractiveWindowTests.cs

```diff
@@ -726,6 +726,17 @@ public void CancelMultiLineInput()
             Assert.Equal("> ", snapshot.GetText());
         }
 
+        [Fact]
+        public void SelectAllInHeader()
+        {
+            Window.WriteLine("Header");
+            Window.FlushOutput();
+            Assert.Equal("Header\r\n> ", Window.TextView.TextBuffer.CurrentSnapshot.GetText());
+
+            Window.TextView.Caret.MoveTo(new SnapshotPoint(Window.TextView.TextBuffer.CurrentSnapshot, 1));
+            Window.Operations.SelectAll(); // Used to throw.
```

Comment:
What should be selected at this point? Perhaps add Assert for selected range or content.